modules/form-field/css/stories/stories_TextArea.tsx

```diff
@@ -61,11 +67,9 @@ storiesOf('CSS/Form Field/TextArea/Top Label', module)
             Label
           </label>
           <div className="wdc-form-field">
-            <textarea
-              className="wdc-form-textarea wdc-form-disabled"
-              placeholder="Placeholder"
-              disabled={true}
-            />
+            <div className="wdc-form-textarea wdc-form-disabled">
+              <textarea placeholder="Placeholder" disabled={true} />
```

Comment:
This example has a `label` with an `htmlFor="textarea"`, but there is no `id` here